pkg/scheduler/framework/session.go

```diff
@@ -272,6 +272,10 @@ func jobStatus(ssn *Session, jobInfo *api.JobInfo) scheduling.PodGroupStatus {
 		// If there're enough allocated resource, it's running
 		if int32(allocated) >= jobInfo.PodGroup.Spec.MinMember {
 			status.Phase = scheduling.PodGroupRunning
+			// If all allocated tasks is succeeded, it's completed
+			if len(jobInfo.TaskStatusIndex[api.Succeeded]) == allocated {
```

Comment:
for batchv1 native job, if using .spec.completions and .spec.parallelism in job, for case, successed 10, in the same time, the queue is full, some pod will pending,  len(jobInfo.TaskStatusIndex[api.Succeeded]) == allocated will be true,  job not finished but pg status is completed, is it happend?